src/pages/components/progress-bar/usage.mdx

```diff
@@ -368,16 +368,17 @@ dismissed depending on what is most suitable for the use case.
 
 The active loading state indicates that the action is still in progress.
 
-#### Success
+<Row>
+<Column colLg={12}>
 
-The success loading state indicates that the action completed successfully.
+![Progress bar as a part of a page](images/progress-bar-usage-19.png)
```

Comment:
I'm not sure moving this image up works here. It's very big for the context. If we want an example after each state, I'd suggest, smaller cropped versions. And perhaps a table somewhere so you can see all the definitions in one view without scrolling so much